### ISSUE TITLE
install older version

### ISSUE THREAD

**Issue by loveduah1** [OPM/opm-simulators#5370]
Please, how do I install older versions of OPM?

**Comment by loveduah1** [comment]
Please I unintentionally updated my OPM package to 2024.04 but want to work with 2023.10. Please how do I go a generation back.

**Comment by blattms** [comment]
Which operating system?

**Comment by loveduah1** [comment]
Hello Sir,

please Linux (wsl)

**Comment by akva2** [comment]
launchpad only hosts newest packages.

**Comment by loveduah1** [comment]
Please does that mean I can’t go back? The issue is I didn’t even know I was upgrading. My customized programs don’t work now.

**Comment by akva2** [comment]
yes, you'd have to build 2023.10 yourself, unless you have the packages cached somewhere..

**Comment by loveduah1** [comment]
Thank you

**Comment by loveduah1** [comment]
Please is there a way to get the cumulative injection/production report in the FILE.PRT (when I run the attached)?
[opm2024.zip](https://github.com/OPM/opm-simulators/files/15403048/opm2024.zip)
